proto/allocation/allocation.proto

```diff
@@ -91,12 +91,18 @@ message AllocationResponse {
   string address = 4;
   string nodeName = 5;
   string source = 6;
+  optional GameServerMetadata metadata = 7;
 
   // The gameserver port info that is allocated.
   message GameServerStatusPort {
     string name = 1;
     int32 port = 2;
   }
+
+  message GameServerMetadata {
```

Comment:
Again, same structure as `MetaPatch` - should we reuse it?

Or do we have the concern the we can't expand if we need to later?

Reply:
That's the primary concern. I did think about doing exactly that, but doing so would cause issues with back/forwards compatibility if someone wanted to change either of them. They're semantically different enough that it seemed risky.

Reply:
That makes sense. I rescind my question then 👍🏻